src/XRP/shared/rippled-web-socket-schema.ts

```diff
@@ -276,21 +316,24 @@ interface CurrencyValuePair {
 type EmptyObject = Record<never, never>
 
 export {
+  WebSocketReadyState,
+  RippledMethod,
   WebSocketRequest,
+  SubscribeRequest,
+  AccountLinesRequest,
+  GatewayBalancesRequest,
+  AccountOffersRequest,
   WebSocketResponse,
+  ResponseStatus,
   WebSocketFailureResponse,
   StatusResponse,
   TransactionResponse,
   AccountLinesResponse,
   AccountLinesSuccessfulResponse,
   GatewayBalancesResponse,
   GatewayBalancesSuccessfulResponse,
-  WebSocketReadyState,
-  RippledMethod,
-  ResponseStatus,
-  SubscribeRequest,
-  AccountLinesRequest,
-  GatewayBalancesRequest,
+  AccountOffersResponse,
+  AccountOffersSuccessfulResponse,
```

Comment:
why did the order of these exports change? not a big deal, but messes with our commit history

Reply:
I just reordered them so the order made more sense, instead of being really messy.